docs/tutorials/zero-to-hero.md

```diff
@@ -4,22 +4,13 @@ title: Zero to Hero: Writing an Oracle
 sidebar_label: Zero to Hero
 ---
 
-<blockquote class="danger">
-<strong>heads up</strong><br><br>
-
-We are **currently migrating away from NEAR Studio** to a better experience for developers.  This article includes references to NEAR Studio which is being phased out.
-
-For the most up to date examples of building on the NEAR platform, please refer to https://examples.nearprotocol.com
-
-</blockquote>
-
 ## Tutorial Overview
 
 In this tutorial, you'll quickly get up to speed with building on the NEAR Protocol. You'll learn to build a simple [Oracle](https://cryptobriefing.com/what-is-blockchain-oracle/) that can query external APIs and provide this data to the blockchain.
 
 Because blockchains are closed systems. Smart contracts can only interact with data that lives on the blockchain. They cannot natively interface with data in the external world. Thus an Oracle is necessary to connect the blockchain with the outside world. There are various types of [Oracles](https://cryptobriefing.com/what-is-blockchain-oracle/). We'll be implementing the most basic one - an Inbound Oracle that provides smart contracts with data from the external world.
 
-Once we've built the Oracle, we'll finish this tutorial with a real world example where we get the price of Bitcoin from CoinDesk, and then create a simple "Wager" contract that pays out the winner between two gamblers who have bet on the "over-under" price of Bitcoin.
+Once we've built the Oracle, we'll finish this tutorial with a real world example where we get the price of Bitcoin from CoinDesk and saves it to NEAR blockchain.
 
 **Tutorial Pre-requisites:**
```

Comment:
again, wasn't part of this PR, but there's no hyphen in prerequisite:
https://www.merriam-webster.com/dictionary/prerequisite

Reply:
fixed